### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by mermop**
Basically copied wholesale from https://github.com/ServiceInnovationLab/RapuTure/pull/122

Configure the application for Docker, so that it can be run in containers :whale:

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,8 +28,8 @@ before_install:
 before_script:
 - bundle install
 - yarn install
-- RAILS_ENV=test bundle exec rake db:create db:migrate
-- RAILS_ENV=test bundle exec rake assets:precompile
+- RAILS_ENV=test DATABASE_URL=postgresql://postgres@localhost:5433 bundle exec rake db:create db:migrate
+- RAILS_ENV=test DATABASE_URL=postgresql://postgres@localhost:5433 bundle exec rake assets:precompile
```
Comment (Br3nda) [inline .travis.yml]: can this go in a env piece?
eg.

```
env:
  global:
    DATABASE_URL="postgresql://postgres@localhost:5433"
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.6
```
Comment (Br3nda) [inline Dockerfile]: is there any clever way to read the version from `.ruby-version`, so we don't need to maintain in multiple places.

File: .travis.yml (hunk 1 above)
Comment (mermop) [reply .travis.yml]: Probably, I'll move it once it's working...

File: Dockerfile (hunk 2 above)
Comment (mermop) [reply Dockerfile]: Could go in a .env file?

Comment (Br3nda) [reply Dockerfile]: we can read it from 
`export RUBY_VERSION=`cat .ruby-version`

**Comment by mermop** [comment]
Closing this - can be a starting point if the team decides Docker is a good idea in the future, but it doesn't seem super necessary right now

**Pull request closed without merging**